sql/moz-fx-data-shared-prod/glam_etl/glam_fenix_release_aggregates/metadata.yaml

```diff
@@ -0,0 +1,4 @@
+workgroup_access:
```

Comment:
It's not obvious to me that `glam_fenix_release_aggregates` is being created anywhere. `glam_etl.glam_fenix_release_aggregates_v1` exists in both projects.

Reply:
Thanks. I just spotted this. Bad copy/paste while working on https://github.com/mozilla/bigquery-etl/pull/5020 in parallel.